Enable tests that were beeing skipped in CI

lucas-p-bernardes opened the pull request.

lucas-p-bernardes committed:
tests/integration: Enable tests that access localhost network in CI

These tests were failing because the container running the jobs in CI does not
have the --network=host configuration. Therefore, it couldn't access the host's
network. To solve this, the commands that tried to download files from servers
created with torizoncore-builder were adapted to run in a busybox container
with the --network=host configuration.

Related-to: TCB-503

Signed-off-by: Lucas Bernardes <lucas.bernardes@toradex.com>

lucas-p-bernardes committed:
tests/integration: Enable plaform tests that were being skipped in CI

The jobs for platform tests are already beeing done separately in CI.
Since the reason for skip this was avoid doing this tests in parallel,
this was aready solved and it's possible to enable again.

Related-to: TCB-503

Signed-off-by: Lucas Bernardes <lucas.bernardes@toradex.com>

lucas-p-bernardes committed:
aval-tests: Change default AVAL machines to speed up tests

Since AVAL lock a device, finish the tests and then release the device, it's
not possible to run AVAL tests with same device in parallel. This commit
change the default machines to run only one job per machine, this way it's
possible to speed up the tests and cover more machines.

Signed-off-by: Lucas Bernardes <lucas.bernardes@toradex.com>

lucas-p-bernardes commented:
Pipeline: https://gitlab.int.toradex.com/lucas.bernardes/torizoncore-builder/-/pipelines/110362

rborn-tx commented on tests/integration/aval/aval-tests.yml:
```diff
@@ -65,6 +65,8 @@ build-aval-docker:
 # AVAL_TEST_ALL can be defined as "true" on a Gitlab schedule to run tests on all devices
 aval-apalis-imx6q-kirkstone-release:
   extends: .aval-template
+  rules:
+    - if: '$AVAL_TEST_ALL =~ /^true$/i'
```
I'd suggest adding this variable to the "variables" section with a proper description (as I did on this [PR](https://github.com/torizon/torizoncore-builder/pull/50)).

lucas-p-bernardes replied:
It's done!

rborn-tx reviewed (approved):
I've added a suggestion regarding a variable that already existed. Apart from that, looks good to me.

lucas-akira reviewed (approved):
LGTM

lucas-p-bernardes committed:
aval-tests: Define AVAL_TEST_ALL in variables area, as false by default

This fit the standard pipeline variables definition in
torizoncore-builder repo.

Signed-off-by: Lucas Bernardes <lucas.bernardes@toradex.com>
